We are not using certifi package on OpenIndiana as the location of the standard
certs bundle is patched directly into requests.

--- requests-2.27.1/setup.py.orig	2022-01-17 20:16:34.683126698 +0000
+++ requests-2.27.1/setup.py	2022-01-17 20:17:10.861695756 +0000
@@ -46,7 +46,6 @@
     'idna>=2.5,<3; python_version < "3"',
     'idna>=2.5,<4; python_version >= "3"',
     'urllib3>=1.21.1,<1.27',
-    'certifi>=2017.4.17'
 
 ]
 test_requirements = [
